unittest/libmariadb/errors.c

```diff
@@ -196,7 +196,7 @@ static int test_cuted_rows(MYSQL *mysql)
   check_mysql_rc(rc, mysql);
 
   count= mysql_warning_count(mysql);
-  FAIL_UNLESS(count == 2, "warnings != 2");
+  FAIL_UNLESS(count == 1, "warnings != 1");
```

Comment:
I verified looking at the packet and it returns only 1 warning. This is in mysqldb.
It returns 2 if it is mariadb (and therefore skipped in mariadb-3 for mysql [here](https://github.com/mariadb-corporation/mariadb-connector-c/blob/9c84958266a7fc61cd5b31b623cf4e26c3320713/unittest/libmariadb/errors.c#L179-L180)